serverless/src/util/urs/getAccessTokenFromJwtToken.js

```diff
@@ -0,0 +1,26 @@
+import { getDbConnection } from '../database/getDbConnection'
+import { getVerifiedJwtToken } from '../getVerifiedJwtToken'
+
+// Knex database connection object
+let dbConnection = null
```

Comment:
I'm not convinced this provides us any value, does this value get kept in memory with this scope or are we just overwriting a potentially 'cached' value each time we call this method?